### DISCUSSION TITLE
Change refresh rate of spectrum display

### DISCUSSION THREAD

**Discussion by Shallrath** [wwmm/easyeffects#1153] [Ideas]
Hi,
I think the spectrum is extremely annoying when being refreshed only around 2..5 Hz or whatever this looks like. It causes a very distracting flickering so I turned it off. Maybe the refresh rate could be made adjustable or just set to a default of 30 or 60 Hz?

**Comment by wwmm** [comment]
It is a known issue for which I don't have a solution yet. Take a look at https://github.com/wwmm/easyeffects/issues/1001
